docker-compose.yml

```diff
@@ -60,15 +60,15 @@ services:
       - ./:/code
     environment:
       DATABASE_HOST: postgres
-      BROKER_URL: amqp://guest:guest@rabbitmq:5672/
-      SHARE_API_URL: http://web:8000/
-      ELASTICSEARCH_URL: http://elasticsearch:9200/
+      BROKER_URL: amqp://guest:guest@rabbitmq:35672/
```

Comment:
Wouldn't this need to be kept as the default port? The actual connection should be happening inside the container.